src/VisualStudio/CSharp/Impl/LanguageService/CSharpCodeCleanupFixer.cs

```diff
@@ -309,6 +311,10 @@ private async Task<bool> FixTextBufferAsync(TextBufferCodeCleanUpScope textBuffe
                 cancellationToken = cancellationTokenSource.Token;
 
                 var buffer = textBufferScope.SubjectBuffer;
+
+                // Load document here so that the partial load message actually shows up before the next scope kicks in.
+                var document = await buffer.CurrentSnapshot.GetFullyLoadedOpenDocumentInCurrentContextWithChangesAsync(context.OperationContext).ConfigureAwait(false);
```

Comment:
:exclamation: This method uses JTF for switching contexts (line 353, 355), so it needs to use `ConfigureAwait(true)`

```suggestion
                var document = await buffer.CurrentSnapshot.GetFullyLoadedOpenDocumentInCurrentContextWithChangesAsync(context.OperationContext).ConfigureAwait(true);
```